### ISSUE TITLE
change font / no italics

### ISSUE THREAD

**Issue by mbalch** [sheodox/alexandrite#84]
I'm loving alexandrite on Lemmy, it's so much better than the stock UI except I can't figure out how to set the default display font to normal instead of italics.  I'm not sure if this is a feature request or a bug or maybe I have something misconfigured.  Lemmy without alexandrite displays in normal font, no italics.  Thanks!

**Comment by sheodox** [comment]
Thanks!

My guess is you might have a font installed but are missing the font variant that the site is trying to use so it falls back to the italics, because almost nothing on the site should be in italics. What OS and browser are you using?

**Comment by mbalch** [comment]
Win10/Chrome (latest build 118.0...).   It looks like Cambria Math ??  is the standard font.  I don't recall ever changing it.  What should it be?  Thank you.

**Comment by sheodox** [comment]
I'm not sure, but I want to see if that's actually the one in use. Can you try this:

1. right click on some italic text and click "Inspect", dev tools will open
2. somewhere you should see some tabs like "Styles, Computed, Layout, Event Listeners", click on "Computed"
3. what font does it say at the bottom of that tab under "Rendered Fonts"

![image](https://github.com/sheodox/alexandrite/assets/3468630/287e94f5-cb5d-4c2e-afcd-db204ff3563a)

The fonts Alexandrite uses are `-apple-system, "Noto Sans", Roboto, Ubuntu, "Segoe UI", sans-serif` (the first one available on your system is used). I think `Segoe UI` is a font included with Windows so I'd have expected that to be in use instead of falling back to your default Cambria Math, so I'm wondering if at some point you tried installing Noto Sans but only installed the italic font variants.

**Comment by mbalch** [comment]
Here's what I was seeing:

![image](https://github.com/sheodox/alexandrite/assets/556328/98e1978c-1438-4cc0-af5b-decc5ea4c10b)

I checked the Windows Font folder and only had Noto Sans Italic installed, so I uninstalled that particular font. Reloading a.lemmy.world rendered the page without italics using Segoe-UI

![image](https://github.com/sheodox/alexandrite/assets/556328/44685d19-53f8-4319-a3e4-ec5f99af1ea8)

I suppose alternately I could install the Noto Sans family from google but this works for me.

Thank you so much for taking the time to help me troubleshoot this.  Alexandrite looks fantastic now.  Cheers!
